src/config/_inputs.scss

```diff
@@ -0,0 +1,25 @@
+
+// Dependencies
+@import 'config/variables';
+@import 'config/colors';
+@import 'config/fonts';
+
+$padding: map-get($variables, 'padding');
+
+// Declarations
+@mixin inputs() {
+  color: $color-base-black;
+  padding: .75 * map-get($padding, 2);
+  font-size: 18px;
+  border: 1px solid $color-secondary-grey;
+
+  &::placeholder {
+    color: $color-secondary-grey;
```

Comment:
Color should be inherit... if we want to differentiate from user inputted text, using opacity would be better.